sdk/eventhub/Azure.Messaging.EventHubs/assets/functions/role-assignment-functions.ps1

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+function AssignRole()
```

Comment:
Isn't role assignment just another type of resource management?   I wonder if this should be merged into the resource script.